doc/examples/xx_applications/plot_rank_filters.py

```diff
@@ -44,11 +44,15 @@
 noisy_image = img_as_ubyte(data.camera())
```

Comment:
@ahojnnes Thanks for review! Yes, I agree. Titles become too small in `sphinx-gallery`.
I'll give the PR an another round and will notify you on results.